index.ts

```diff
@@ -2,15 +2,26 @@ import bodyParser from 'body-parser';
 import frameguard from 'frameguard';
 import cors from 'cors';
 import { graphqlHTTP } from 'express-graphql';
+import rateLimit from 'express-rate-limit';
 import api from './server';
 import { schema, rootValue } from './server/graphql';
 import defaultQuery from './server/graphql/examples';
+import { sendError } from './server/helpers/utils';
 
 export default app => {
   app.use(bodyParser.json({ limit: '20mb' }));
   app.use(bodyParser.urlencoded({ limit: '20mb', extended: false }));
   app.use(frameguard({ action: 'deny' }));
   app.use(cors());
+  app.set('trust proxy', 1);
+  app.use(rateLimit({
+    windowMs: 30 * 1000,
+    max: 12,
```

Comment:
I think 12 might be too low for 30s (you might open 10 tabs, for example). What do you think about making the window a bit shorter, e.g. 10s?

I think what we're seeing are very large, sudden bursts of requests from programmatic sources, so the shorter the window the less normal users could be impacted by it.

Reply:
I changed this manually just to test a deployment on Heroku. The initial value was 120, i think 12/10 will not be enough, some page in the UI will need multiple request to the API, and we want to allow script to gather data too, we can start with high amount and see how the hub handle it, then lower if needed.